TrainingApp/Controller/SettingsViewController.swift

```diff
@@ -119,6 +174,87 @@ final class SettingsViewController: UIViewController {
     
     // MARK: - Helpers
     
+    private func checkThemeSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_THEME)) {
+            darkModeSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.DARK_THEME, forKey: self.THEME_KEY)
+        }
+        else {
+            darkModeSwitch.setOn(false, animated: true)
+            self.userDefaults.set(self.LIGHT_THEME, forKey: self.THEME_KEY)
+        }
+    }
+    
+    private func checkNotificationSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_NOTIFICATION)) {
+            notificationsSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.NOTIFICATION_ON, forKey: self.NOTIFICATION_KEY)
+            checkFlightSwitchState()
+            checkExecutiveSwitchState()
+            checkDiscountSwitchState()
+        }
+        else {
```

Comment:
remove this space
make it 
if {
} else {
}

Reply:
Done 👍